spec/rails_helper.rb

```diff
@@ -1,12 +1,14 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'test'
 require File.expand_path("../../config/environment", __FILE__)
+require 'rails_helper'
```

Comment:
it looks like this is requiring rails_helper, but the file was also renamed, so it's requiring itself?